languoids/tree/afro1255/egyp1245/egyp1246/md.ini

```diff
@@ -13,6 +13,7 @@ countries =
 links = 
 	https://www.wikidata.org/entity/Q50868
 	https://en.wikipedia.org/wiki/Egyptian_language
+timespan = -3000-01-01/-3000-01-01
```

Comment:
Can interval `start` and `end` be identical for this attribute (more occurrences below)?